app/src/main/java/ciserver/PushEvent.java

```diff
@@ -26,6 +26,9 @@ class Commit {
 	ArrayList<String> modified; // modified files
 }
 
+/**
+ * This class represents the relevant information from a Push Event to GitHub
+ */
 public class PushEvent {
 	String ref; // hold branch name
 	String compare; // link to see changes
```

Comment:
Should probably document the 6 fields of the class as well since they are very relevant to the API.

Reply:
I used the constructor to document this (JavaDoc with `@param`s), could that be an option here as well?

Reply:
Well there is no constructor for `PushEvent` since it simply specifies which values are parsed from JSON by the GSON library.

Reply:
I tried to solve this now by just adding a link to the GH documentation for the fields.